profiles/default/linux/amd64/17.0/musl/make.defaults

```diff
@@ -0,0 +1,10 @@
+# Copyright 1999-2015 Gentoo Foundation.
+# Distributed under the terms of the GNU General Public License v2
+
+CHOST="x86_64-gentoo-linux-musl"
+
+# Anthony G. Basile <blueness@gentoo.org> (01 Jul 2014)
+# Multilib-related setup, bug #515130
+MULTILIB_ABIS="amd64"
```

Comment:
Doesn't no-multilib profile set that? Or does some other profile override it?

Reply:
I had to force it.